.github/chainguard/lifecycle-eol-cve-triage.sts.yaml

```diff
@@ -1,7 +1,8 @@
 issuer: https://accounts.google.com
 
 # prod-images: bot-triageolwolfi@staging-enforce-cd1e.iam.gserviceaccount.com
-subject: "108215247871814150708"
+# bot-triageolwolfi@prod-enforce-fabc.iam.gserviceaccount.com
+subject: "(102425108416137555121|108215247871814150708)"
```

Comment:
does subject work for multiple  items 
or it should be subject_pattern ?

Reply:
good call! I have to use subject_pattern here 👍🏻